kore/src/main/scala/org/kframework/unparser/Unparse.scala

```diff
@@ -18,64 +19,65 @@ object ToKast {
     unparse(s => b ++= s, false, 0, k)
     b.toString()
   }
-  def apply(k: K, out: PrintStream): Unit = {
+  def apply(k: K, out: PrintStream): Unit =
     unparse(out.print, false, 0, k)
-  }
 
   def apply(l: KLabel): String = unparse(false, l)
 
   def escape(s: String): String = StringUtil.enquoteKString(s)
 
-  def unparse(inParen: Boolean, l: KLabel) : String = {
+  def unparse(inParen: Boolean, l: KLabel): String = {
     var name: String = ""
-    if (l.name.matches("[#a-z][a-zA-Z0-9]*")
-        && l.name != "#token" && l.name != "#klabel") {
+    if (
+      l.name.matches("[#a-z][a-zA-Z0-9]*")
+      && l.name != "#token" && l.name != "#klabel"
+    ) {
       name = l.name
     } else if (inParen) {
-      name = " `"+ escapeBackTicksAndSlashes(l.name) +'`'
+      name = " `" + escapeBackTicksAndSlashes(l.name) + '`'
     } else {
       name = '`' + escapeBackTicksAndSlashes(l.name) + '`'
     }
     if (l.params.isEmpty) {
       name
     } else {
-      name + "{"+ l.params.map(_.name).reduce((s1,s2) => s1 + "," + s2) + "}"
+      name + "{" + l.params.map(_.name).reduce((s1, s2) => s1 + "," + s2) + "}"
     }
   }
 
-  def escapeBackTicksAndSlashes(str: String) : String =
+  def escapeBackTicksAndSlashes(str: String): String =
     str.replaceAll("\\\\", "\\\\\\\\").replaceAll("`", "\\\\`");
 
   /**
-   * Recursive worker function for printing KAST terms.
-   * The extra arguments are needed to respect precedence and the lexical syntax.
+   * Recursive worker function for printing KAST terms. The extra arguments are needed to respect
+   * precedence and the lexical syntax.
    *
-   * <ul>
-   * <li>Precedence level 0 is for the top of a term or within an argument list.
-   * <li>Precedence level 1 is for arguments of a KRewrite.
-   * <li>Precedence level 2 is for entries in a KSequence,
-   * </ul>
-   * The only case where braces may be needed is around a KSequence which is
-   * an argument of a KRewrite.
+   * <ul> <li>Precedence level 0 is for the top of a term or within an argument list. <li>Precedence
+   * level 1 is for arguments of a KRewrite. <li>Precedence level 2 is for entries in a KSequence,
+   * </ul> The only case where braces may be needed is around a KSequence which is an argument of a
+   * KRewrite.
```

Comment:
Needs to be reworked here as well, we completely  lost the readability we had before